_posts/2020-09-14-Introduction-to-API-Management.md

```diff
@@ -0,0 +1,292 @@
+	     
+<img src="/img/2020-09-14-Introduction-to-API-Management/WithoutApiManagementTool2.jpg" alt="API landscape without an API management tool" width="350" height="200" class="image fit">
+
+### API landscape with an API management tool
+This all changes when we implement an API Management tool.
+
+<img src="/img/2020-09-14-Introduction-to-API-Management/WithApiManagementTool2.jpg" alt="API landscape with an API management tool" width="500" height="350" class="image fit" style="vertical-align:middle;margin-left:2%"/>       
+
+In this situation an API management tool (API Gateway) is deployed between the client and the APIs.
+All request first have to pass the API management tool, before they are forwarded to the right service.
+If the Gateway is installed in the same organizational structure, it’s not required to have your API’s implementing additional cross cutting functionalities.
+The API gateway will be responsible of performing the necessary checks before allowing the clients to access the Data.
+I do hear you thinking already “We’re not going to install an additional tool, only for splitting out security rules from the API’s", but that’s not the only thing an API management tool can do.
+
+### Responsibilities of an API Management tool
+I listed out the most important responsibilities for you, so that you can understand the power and the actual benefits of an API Management tool:            
+     
+1.	Authentication 
+    * Enables one of the most important parts of security on an API. Validating the person’s identity!  
+    There are different kinds of Authentication:   
+        * Http Basic Authentication
+            * Using this approach, a user agent simply provides a username and password to prove their authentication.  
+            This approach does not require cookies, session ID’s, or login pages because it leverages the HTTP header itself. While simple to use, this method of authentication is vulnerable to attacks that could capture the user’s credentials in transit.
+            
+        * OAuth
+            * OAuth 2.0 is the best choice for identifying personal user accounts and granting proper permissions.  
+            With this method, the user logs into a system. That system will then request authentication, usually in the form of a token. The user will then forward this request to an authentication server, which will either reject or allow this authentication.
+        
+        * OpenID Connect & OAuth 2.0 API
+            * OpenID Connect is a simple identity layer on top of the OAuth 2.0 protocol, which allows computing clients to verify the identity of an end-user based on the authentication performed by an authorization server, as well as to obtain basic profile information about the end-user in an interoperable and REST-like manner.
+        
+        * API Keys
+            * An API key is an identifier meant to identify the origin of web service requests (or similar types of requests).  

```

Comment:
It's just a way to identify the user's identity, not the origin of web service requests

Reply:
oops, I Copied it from the user guide source